libgstd/gstd_return_codes.h

```diff
@@ -129,6 +129,16 @@ enum _GstdReturnCode
    */
   GSTD_MISSING_NAME,
 
+  /**
+   * The system has run out of memory
+   */
+  GSTD_OOM,
+
+  /**
+   * Unknown error
+   */
+  GSTD_UNKNOWN_ERROR,
```

Comment:
why are we adding codes we are not using?

Reply:
My bad. They were used in a previous implementation